packages/plugin-graphql/src/index.js

```diff
@@ -64,12 +64,13 @@ class GraphQLResource extends ResourceInterface {
   async optimize(url, body) {
     return new Promise((resolve, reject) => {
       try {
-        body = body.replace('<head>', `
+        body = body.replace( `
+        
```

Comment:
Hey @Yasir761

What would need here is to just [move the _position_ of `<head>` tag](https://github.com/ProjectEvergreen/greenwood/blob/master/packages/plugin-graphql/src/index.js#L71) to come _before_ the `<script>` block in the replace function.  From what I see here is looking to replace the `<script>` block _with just_ `<head>`.

I think if you run `yarn test` locally it would likely fail the specs.  I think we can even ensure this change by slightly tweaking [all instances of this kind of spec in the package](https://github.com/ProjectEvergreen/greenwood/blob/master/packages/plugin-graphql/test/cases/query-config/query-config.spec.js#L84)
```js
// instead of
const scriptTags = dom.window.document.querySelectorAll('script');

// we should ensure the nesting by doing this to ensure it is within the <head> tag
const scriptTags = dom.window.document.querySelectorAll('head script');
```

Let me know if that all makes sense and here are some [docs](https://github.com/ProjectEvergreen/greenwood/blob/master/.github/CONTRIBUTING.md#testing) on how to get into testing for this project.